shared/templates/sebool/ansible.template

```diff
@@ -4,34 +4,44 @@
 # complexity = low
 # disruption = low
 
+{{% if product in ['ol8', 'ol9', 'rhel8', 'rhel9'] %}}
+{{% set PACKAGE_NAME = "python3-libsemanage" %}}
+{{% elif product == "sle15" %}}
+{{% set PACKAGE_NAME = "policycoreutils" %}}
+{{% else %}}
+{{% set PACKAGE_NAME = "libsemanage-python" %}}
+{{% endif %}}
+
+- name: Ensure {{{PACKAGE_NAME}}} installed
+  package:
+    name: "{{{ PACKAGE_NAME}}}"
+    state: present
+
+{{% if product == "sle15" %}}
+- name: Ensure list of additional packages are installed on SLE 15
+  become: yes
+  package:
+    name:
+      - policycoreutils-python-utils
+      - selinux-tools
+      - python3-selinux
+      - python3-semanage
+    state: present
+{{% endif %}}
+
 {{% if SEBOOL_BOOL %}}
 - name: Set SELinux boolean {{{ SEBOOLID }}} to {{{ SEBOOL_BOOL }}}
   seboolean:
     name: "{{{ SEBOOLID }}}"
     state: "{{{ SEBOOL_BOOL }}}"
     persistent: yes
+  when: ansible_facts.selinux.status == 'enabled'  
```

Comment:
Good test. Something similar should be good in Bash too.

Reply:
I am guessing @marcusburghardt you meant something like this [65acdab](https://github.com/ComplianceAsCode/content/pull/11245/commits/65acdabaffd3a2f3e52f85a2ab6be9e2b4aa41c7) ?

Reply:
Almost there. Here in Ansible the first task ensures the semanage library for all distros. This is not ensured in Bash.

So, besides the updates in https://github.com/ComplianceAsCode/content/pull/11245/commits/65acdabaffd3a2f3e52f85a2ab6be9e2b4aa41c7, we should also ensure the semanage library for all distros in the Bash remediation so Bash and Ansible are aligned. Currently, if there is no semanage library previously installed, the Bash remediation will fail to all other distros except sle15.